thepracticaldev/dev.to
# Omar el khatib cs/fixing scrollbar for small screens 9016

## What type of PR is this? (check all applicable)


## Description

## Related Tickets & Documents

## QA Instructions, Screenshots, Recordings

Go to profile -> resize the window -> you should see annoying scroll bar in the bottom ,
also same for mobile app.

Screenshot after fixing

before

![fix_respo_before](https://user-images.githubusercontent.com/35725554/86118886-5b65be80-ba86-11ea-9f67-df1f1874ceb3.PNG)

after
![fix_respo](https://user-images.githubusercontent.com/35725554/86118745-28bbc600-ba86-11ea-9cf4-4db6378e0b91.PNG)


## Added tests?


## Added to documentation?


## [optional] Are there any post deployment tasks we need to perform?

## [optional] What gif best describes this PR or how it makes you feel?

![alt_text](gif_link)
